### PR TITLE
Catalog ready

### PR DESCRIPTION
**Pull request by MaksymMaliuk**
[DEMO LINK](https://MaksymMaliuk.github.io/layout_catalog/)
[TEST REPORT LINK](https://MaksymMaliuk.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/header.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    &__link.is-active::after {
+      content: "";
+      position: absolute;
+      bottom: 0;
+      width: 100%;
+      height: 4px;
+      background-color: #{$blue-color};
+      border-radius: 8px;
+    }
```
Comment (sergey-lipikhin) [inline src/styles/blocks/header.scss:49]: You can place it under `&__link`. It would be better due to sass nesting capabilities

**Comment by sergey-lipikhin** [comment]
image

I'm not sure you need this bottom-margin for **THE LAST LINE** of cards. In mockup this bottom space not even 48px.

I would recommend you to play with:

image

File: src/styles/blocks/catalog.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.catalog {
+  margin: 0 auto;
```
Comment (sergey-lipikhin) [inline src/styles/blocks/catalog.scss:2]: Invalid BEM syntax. You can not define Block positioning, so margin should be removed.

You can try to create mix: placing all positioning logic of Block (your margin) on Element wrapper.
You can do smth like that:

```
<main class="main">  
  <section class="catalog main__catalog"> <------- here define position rules for .catalog via main__catalog element
```

**Review by sergey-lipikhin — COMMENTED** [review]
Use mix :
https://youtu.be/JOJ-mENrg_s?t=254 (with timing)

**Review by sergey-lipikhin — APPROVED** [review]
GJ

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+    color: #{$main-accent-color};
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
```
Comment (illiarerun) [inline src/styles/blocks/card.scss:53]: better don't do it, it's hard to comprehend and price can be confused with a block

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**